src/main/java/com/carhouse/config/WebConfig.java

```diff
@@ -0,0 +1,56 @@
+package com.carhouse.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.thymeleaf.spring5.templateresolver.SpringResourceTemplateResolver;
+import org.thymeleaf.spring5.view.ThymeleafViewResolver;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com.carhouse")
+public class WebConfig implements WebMvcConfigurer {
+
+    @Bean
+    public SpringResourceTemplateResolver templateResolver() {
+        SpringResourceTemplateResolver templateResolver = new SpringResourceTemplateResolver();
+        templateResolver.setPrefix("/WEB-INF/views/");
+        templateResolver.setSuffix(".html");
+        templateResolver.setTemplateMode("HTML5");
+        templateResolver.setCacheable(false);
+        return templateResolver;
+    }
+
+    @Bean
+    public SpringTemplateEngine templateEngine() {
+        SpringTemplateEngine templateEngine = new SpringTemplateEngine();
+        templateEngine.setTemplateResolver(templateResolver());
```

Comment:
think method parameters injection will be better

Reply:
Ok

Reply:
But this method return template engine and is used by a view resolver to set a template engine,
I think it will be more clear to leave the name of this method 'templateEngine'
By the way, this name is used in all thymeleaf tutorials

Reply:
Did I ask about method renaming?
I propose to refactor it to `templateEngine(SpringResourceTemplateResolver templateResolver ...)`